wishlist/wishlist.css

```diff
@@ -0,0 +1,46 @@
+.wishlist-container {
+    width: 90%;
+    margin: auto;
+}
+
+.wishlist-items-container {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(180px, 1fr));
+    justify-items: center;
+    /* align-items: flex-start; */
```

Comment:
You can remove comments if you don't need them

Reply:
Thanks Aazam for pointing it out, removed it.